app/controllers/users_controller.rb

```diff
@@ -1,9 +1,22 @@
 class UsersController < ApplicationController
   def index
+    @users = Users.all
     puts params
   end
 
```

Comment:
Please add instructions to run the app locally in README. A professional README should have instructions to run the project locally.